tests/go.mod

```diff
@@ -160,8 +161,10 @@ replace (
 	// We always want to test against the latest version of the simapp.
 	cosmossdk.io/simapp => ../simapp
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+	github.com/coinbase/rosetta-sdk-go => github.com/coinbase/rosetta-sdk-go v0.8.2-0.20221007214527-e03849ba430a
```

Comment:
Just curious why we need this import. Can we add a comment here.

Reply:
It is linked to this issue : https://github.com/coinbase/rosetta-sdk-go/issues/449
I have added a comment. Once coinbase make a release we can do change to version v0.8.2.

It is so I can pass `check:spec` like mentioned in the issue. It is however not mandatory.